evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -2746,6 +2746,6 @@ def test_get_component_input_logical_types():
     no_estimator.fit(X, y)
     assert no_estimator.last_component_input_logical_types == {
         "cat": Categorical(),
-        "numeric": Double(),
+        "numeric": Integer(),
```

Comment:
This is a side effect of the change to `Imputer` to keep logical types where possible